src/DependencyInjection/Configuration.php

```diff
@@ -69,6 +69,9 @@ public function getConfigTreeBuilder()
                         ->scalarNode('server')->cannotBeEmpty()->end()
                         ->scalarNode('enabled')->defaultFalse()->end()
                         ->arrayNode('ip_white_list')
+                            ->setDeprecated('The "%node%" option is deprecated. Use "trusted_ip_list" instead with the same values.')
```

Comment:
@greg0ire @franmomu is this enough?

Reply:
For the deprecation it is, sorry, I meant if it was possible to deprecate a parameter in the container since someone can be using the `sonata.user.google.authenticator.ip_white_list` parameter.